### PR TITLE
Container Guide language review

### PR DESCRIPTION
**Pull request by dariavladykina**

### CODE REVIEW

File: container-guide/concepts/containers-bci-app.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
   </info>
   <para>
     Application &bcia;s are &bcia;-Base container images that include specific
```
Comment (dariavladykina) [inline container-guide/concepts/containers-bci-app.xml:22]: The output is _Application SLE BCIs are SLE BCI-Base container images that include specific applications_ and I wonder whether we should reword this to _Application SLE BCIs are SLE Base Container Images that include specific applications_ ?

File: container-guide/concepts/containers-bci-language.xml
Comment (dariavladykina) [inline container-guide/concepts/containers-bci-language.xml:1]: I cannot parse the first sentence in the first para that uses the &bcia; entity:
_Language stack SLE BCIs are built on top of the SLE BCI-Base general-purpose SLE BCI._

Could it be that a &sle; can be used instead of &bcia;?

File: container-guide/concepts/containers-intro.xml [hunk 2]
```diff
@@ -160,8 +160,8 @@
     </orderedlist>
     <para>
       To run a container, you need an image. An image includes all dependencies
-      needed to run the application. For example, the &bcia;-Base image
-      contains the &slea; distribution with a minimal package selection.
+      needed to run the application. For example, the &bcia;-Base image contains
+      the &slea; distribution with a minimal package selection.
```
Comment (dariavladykina) [inline container-guide/concepts/containers-intro.xml:164]: The output _For example, the **SLE BCI-Base image** contains the SLE distribution with a minimal package selection._ confuses me again.

File: container-guide/concepts/containers-verify.xml [hunk 3]
```diff
@@ -48,7 +48,7 @@
     <para>
       To verify a &bcia;, run Cosign in the container. The command below
       fetches the signing key from the &suse; server and uses it to verify the
-      latest &bcia;-Base container image.
+      latest &bcia;-Base container image.
```
Comment (dariavladykina) [inline container-guide/concepts/containers-verify.xml:51]: The output is _[...] verify the latest SLE BCI-Base container image._ 
Maybe use the &sle; &bci; combination here instead?

**Comment by dariavladykina** [comment]
Hi Dima, please take a look at some suggestions for this guide. I am unsure about the _BCI-Base container images_  and have tried to point out every such occurrence in the text, just ignore them if this wording is correct :-) Thanks!

**Comment by dmpop** [comment]
I've added all the suggestions manually, so I'm closing the PR. Thank you very much for your review!
